src/plugins/intel_gna/tests/unit/transformations/gather_sinking_test_utils.hpp

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) 2023 Intel Corporation
```

Comment:
Functions in this file have already been added in gather sinking PR, please watch out not to duplicate and also follow the comments in the other PR.

Reply:
This functions are needed in current PR and cannot been built without it. Another PR is not merged into master.